doc/reference/autoinstall-reference.rst

```diff
@@ -432,52 +435,102 @@ In the new format, the ``primary`` section expects a list of mirrors, which can
 * The special ``country-mirror`` value
 * A mapping with the following keys:
 
-  * ``uri``: The URI of the mirror to use, e.g., ``http://fr.archive.ubuntu.com/ubuntu``.
-  * ``arches``: An optional list of architectures supported by the mirror. By default, this list contains the current CPU architecture.
+  * ``uri`` (Required): The URI of the mirror to use, e.g., ``http://fr.archive.ubuntu.com/ubuntu``.
+  * ``arches`` (Optional): A list of architectures supported by the mirror. By default, this list contains the current CPU architecture.
+
+The URI for the archive mirror does not have to be a country mirror, although it may be the most convenient, and can take the URL of any valid Ubuntu mirror. A list of all registered archive mirrors can be found on `Launchpad <https://launchpad.net/ubuntu/+archivemirrors>`_.
+
+Examples:
+
+.. code-block:: yaml
+
+  # Use custom mirrors for all architectures.
+  autoinstall:
+    apt:
+      mirror-selection:
+        primary:
+          - uri: "$CUSTOM_MIRROR_1"
```

Comment:
I think using URLs (they don't have to exist) will be more useful in an example.

```diff
- uri: "$CUSTOM_MIRROR_1"
+ uri: http://mirror1.internal/ubuntu
```

Wdyt?

Reply:
Yeah that's much better, thanks!